### PR TITLE
[ios] Fixed route restoring.

### PR DESCRIPTION
**Pull request by VladiMihaylenko**
https://trello.com/c/pVkRktmM/248--

### CODE REVIEW

File: iphone/Maps/Classes/CustomViews/MapViewControls/MWMMapViewControlsManager.mm [hunk 1]
```diff
@@ -266,6 +266,16 @@ - (void)setupBestRouter
     f.SetRouter(f.GetBestRouter(self.routeSource.Point(), self.routeDestination.Point()));
 }
 
+- (void)restoreRouteTo:(const m2::PointD &)to
+{
+  auto & f = GetFramework();
+  m2::PointD const myPosition = [MapsAppDelegate theApp].m_locationManager.lastLocation.mercator;
+  f.SetRouter(f.GetBestRouter(myPosition, to));
+  self.routeSource = MWMRoutePoint(myPosition);
+  self.routeDestination = {to, @"Destionation"};
```
Comment (ExMix) [inline iphone/Maps/Classes/CustomViews/MapViewControls/MWMMapViewControlsManager.mm]: s / Destionation / Destination

**Comment by igrechuhin** [comment]
LGTM

**Comment by rokuz** [comment]
LGTM

**Comment by ExMix** [comment]
LGTM

**Pull request merged**